decidim-core/app/types/decidim/core/participatory_space_link_type.rb

```diff
@@ -2,22 +2,21 @@
 
 module Decidim
   module Core
-    ParticipatorySpaceLinkType = GraphQL::ObjectType.define do
-      name "ParticipatorySpaceLink"
+    class ParticipatorySpaceLinkType < Decidim::Api::Types::BaseObject
+      # name "ParticipatorySpaceLink"
```

Comment:
Shouldn't this be uncommented?

```suggestion
      graphql_name "ParticipatorySpaceLink"
```

Reply:
I have removed that comment. GraphQL assigns the class name (without Type) as default type name.